.github/workflows/pull-request-snapshot-release.yml

```diff
@@ -19,20 +19,22 @@ jobs:
         timeout-minutes: 30
 
         steps:
-            - uses: actions/checkout@v2
+            - uses: actions/checkout@v3
               with:
                   ref: ${{ github.head_ref }}
 
             - uses: ./.github/actions/setup
 
             - name: Release with Gradle
               run: |
-                  BRANCH_NAME=PR-${{ env.PR_NUMBER }}
-                  sed -i '/version=/ s/-SNAPSHOT/-'"$BRANCH_NAME"'-SNAPSHOT/' ./gradle.properties
-                  ./gradlew build publishAllVersions -Pzowe.deploy.username=$ARTIFACTORY_USERNAME -Pzowe.deploy.password=$ARTIFACTORY_PASSWORD -Partifactory_user=$ARTIFACTORY_USERNAME -Partifactory_password=$ARTIFACTORY_USERNAME -PpullRequest=$BRANCH_NAME
+                  PR_NUMBER=PR-${{ env.PR_NUMBER }}
+                  sed -i '/version=/ s/-SNAPSHOT/-'"$PR_NUMBER"'-SNAPSHOT/' ./gradle.properties
+                  ./gradlew build publishAllVersions -Pzowe.deploy.username=$ARTIFACTORY_USERNAME -Pzowe.deploy.password=$ARTIFACTORY_PASSWORD -Partifactory_user=$ARTIFACTORY_USERNAME -Partifactory_password=$ARTIFACTORY_USERNAME -PpullRequest=$PR_NUMBER
               env:
                   ARTIFACTORY_USERNAME: ${{ secrets.ARTIFACTORY_USERNAME }}
                   ARTIFACTORY_PASSWORD: ${{ secrets.ARTIFACTORY_PASSWORD }}
+                  BRANCH_NAME: ${{ github.ref_name }}
```

Comment:
Not sure whether you wanted to include the build info changes here too. If that's the case, you're missing other workflows and also the gradle tasks i.e. `packageApiCatalog.doFirst`

Reply:
it was not my intention to fix this, I've just aligned this file with v2 version, but it is a good idea to fix this, I will open another PR